### PR TITLE
Feature/Withdraw custom token - DAO-666

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- Added add custom token button on withdraw token menu
- Added custom token flow in withdraw, complete with validation

### CODE REVIEW

**Comment by Fabricevladimir** [comment]
One thing I've noticed is that with this PR, new deposit and new withdraw have become almost exactly the same. At some point it might be worth looking into combining the two into one process.

**Review by RakeshUP — APPROVED** [review]

File: packages/web-app/src/containers/depositForm/index.tsx [hunk 1]
```diff
@@ -218,12 +218,13 @@ const DepositForm: React.FC = () => {
               validate: addressValidator,
             }}
             render={({
-              field: {name, onBlur, onChange, value},
+              field: {name, onBlur, onChange, value, ref},
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/depositForm/index.tsx:222]: Could you briefly explain what passing the ref here does?

File: packages/web-app/src/containers/configureWithdraw/index.tsx [hunk 2]
```diff
@@ -4,62 +4,168 @@ import {
...
+          setValue('tokenSymbol', data.symbol);
+          setValue('tokenImgUrl', data.imgUrl);
+        } else {
+          const {name, symbol} = await getTokenInfo(tokenAddress, provider);
+          setValue('tokenName', name);
+          setValue('tokenSymbol', symbol);
+        }
+        setValue('tokenBalance', balance);
+      } catch (error) {
+        /**
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/configureWithdraw/index.tsx:77]: Ahm, ok. But what happens if an API (call) fails?

**Review by WissenIstNacht — APPROVED** [review]
Hey thanks!

PR looks ok to me. (Please at least drop a console.error in the catch blocks, the empty catch blocks really keep me up at night haha)

File: packages/web-app/src/containers/depositForm/index.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/containers/depositForm/index.tsx:222]: I needed to be able to focus the custom token address field once the "Add custom token" button is clicked and the menu is closed.

File: packages/web-app/src/containers/configureWithdraw/index.tsx (hunk 2 above)
Comment (Fabricevladimir) [reply packages/web-app/src/containers/configureWithdraw/index.tsx:77]: No really, this one is completely redundant to the point of needing to be removed even lol. The api functions catch their own errors and prints to the console. I don't even know if I like that.

Comment (RakeshUP) [reply packages/web-app/src/containers/configureWithdraw/index.tsx:77]: We could later on add some service like sentry here.

Comment (WissenIstNacht) [reply packages/web-app/src/containers/configureWithdraw/index.tsx:77]: Well then... 😅

**Pull request merged**